deploy.go

```diff
@@ -194,6 +196,28 @@ func RunDeploy(c *cli.Context) error {
 	return nil
 }
 
+func displayNewCommits(owner string, repo string, c *cli.Context, client *github.Client) {
+	ref := Ref(c.String("ref"), git.Head)
+
+	opt := &github.DeploymentsListOptions{
+		Environment: c.String("env"),
+	}
+
+	deployments, _, error := client.Repositories.ListDeployments(owner, repo, opt)
```

Comment:
Same comment as mike. It's idiomatic in Go to handle the error first, and return so the happy path is un-nested:

```go
if err != nil {
        return err
}

// Happy path
```

I'm curious though, does this actually compile with `error == nil`?